utils/Command/MissingInSetCommand.php

```diff
@@ -38,6 +39,9 @@ final class MissingInSetCommand extends Command
         TypedPropertyFromStrictConstructorReadonlyClassRector::class,
         BinarySwitchToIfElseRector::class,
         CountOnNullRector::class,
+
+        // namespace to be moved from CodeQuality to PHPUnit100
+        RemoveSetMethodsMethodCallRector::class,
```

Comment:
@TomasVotruba this is added here temporary due to rector-phpunit PR for move `RemoveSetMethodsMethodCallRector` from phpunit code-quality to phpunit 100

- https://github.com/rectorphp/rector-phpunit/pull/293

to avoid notice:

```bash
bin/rector missing-in-set --ansi

We could not find there rules in configs
========================================

 * vendor/rector/rector-phpunit/config/sets/phpunit-code-quality.php

 * Rector\PHPUnit\CodeQuality\Rector\MethodCall\RemoveSetMethodsMethodCallRector
```